x-pack/plugin/redact/src/main/java/org/elasticsearch/xpack/redact/RedactProcessor.java

```diff
@@ -55,6 +55,12 @@ public class RedactProcessor extends AbstractProcessor {
     private static final String DEFAULT_REDACTED_START = "<";
     private static final String DEFAULT_REDACTED_END = ">";
 
+    protected static final String REDACT_KEY = "_redact";
+    protected static final String IS_REDACTED_KEY = "_is_redacted";
+    protected static final String METADATA_PATH_REDACT = IngestDocument.INGEST_KEY + "." + REDACT_KEY;
+    // indicates if document has been redacted
+    protected static final String METADATA_PATH_REDACT_IS_REDACTED = METADATA_PATH_REDACT + "." + IS_REDACTED_KEY;
```

Comment:
Can we condense these down into one constant? It's a little hard to see exactly what the final key is.

Reply:
Some of these key constants are used in tests, and I didn't want to hard code the constants in other places, that's why I had to define them separately. 

If it helps, I can add a comment for the exact key `_ingest._redact._is_redacted`